# This file contains the configuration for the node

# General parameters
common:
  topics:
    input:
      points: "/velodyne_points"
    output:
      ground: "/ftfcd/debug/ground"
      clusters: "/ftfcd/debug/clusters"

manager:
  publish_debug: true

dbscan:
  octreeResolution: 10
  eps: 5
  minPtsAux: 3
  minPts: 3